### PR TITLE
added btcEurRate gauge

### PR DESCRIPTION
**Pull request by esille**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -162,8 +166,15 @@ async function refreshMetrics() {
   } catch (e) {
     console.log("there was an error on request3 ", e)
   }
+try {
+  const rawResponse4 = await nhClient.getExchangeRates()
+  const data4 = rawResponse4.data
+  //console.log(data4)
+  btcEurRate.set(+data4['BTCEURS'])
```
Comment (tg44) [inline app.js:173]: Can't we just move line 173 under line 165? And then we don't need the whole request block.

Comment (esille) [reply app.js:173]: I'm not javascript expert, so it's definitely better/easier way to implement. Would be also great to have rates as variable filled from docker enviroment to be flexible and not hardcoded.

**Comment by tg44** [comment]
Can you please check the ["Allow edits from maintainers."](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork)?

**Comment by tg44** [comment]
NVM. I pushed it to `main` after a quick edit. Note, that I modified the name of the rates (to make it more generic, I added notes to the readme about it).

**Pull request closed without merging**